pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[build-system]
+requires = ["setuptools>=67.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "sinol-make"
+version = "0.0.1"
+authors = [
+  { name="Mateusz Masiarz", email="m.masiarz@fri.edu.pl" },
+  { name="Tomasz Nowak", email="tomek.nowak.gm.16@gmail.com" }
+]
+description = "Tool for creating sio2 tasks"
+readme = "README.md"
+requires-python = ">=3.7"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+]
+dependencies = [
+	"argparse",
+	"requests",
+  "PyYAML",
+]
+
+[project.urls]
+"Homepage" = "https://github.com/sio2project/sinol-make"
+"Bug Tracker" = "https://github.com/sio2project/sinol-make/issues"
+
+[project.scripts]
+sinol-make = "sinol_make:main"
```

Comment:
What this field means (I have no experience with `pip`)? Does it define which function to execute when used as a CLI tool?

Reply:
It means that when used as CLI it will run function `main` from `__init__.py` in root of sinol-make

Reply:
I see, thanks!